scripts/release.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -e
+
+REPOSITORY="${REPOSITORY:-git@github.com:etcd-io/etcd-operator.git}"
+REGISTRY="${REGISTRY:-gcr.io/etcd-development/etcd-operator}"
+CONTAINER_TOOL="${CONTAINER_TOOL:-docker}"
+
+function main {
+  if [ -z "$1" ]; then
+    echo "Usage: $0 <release version>"
```

Comment:
can we add examples to clarify that both `release.sh v0.1.0` and `release.sh 0.1.0` work?

Reply:
Done.